### ISSUE TITLE
Not able to pull from private repositories on Docker Hub

### ISSUE THREAD

**Issue by davidgiesberg** [twitter-fabric/galley#24]
I have a docker config that looks like this:

````json
{
	"auths": {
		"https://index.docker.io/v1/": {
			"auth": "SECRET",
			"email": "david@employer.com"
		}

	}
}
````

For some reason, I can pull images with `docker pull` but if I try to do the same with `galley pull`, I get errors. I _think_ that https://github.com/twitter-fabric/galley/blob/7cd05cff6f654ca3070fc6982ef84ba1942bd4ad/lib/lib/docker_utils.coffee#L96-L99 is not properly setting the `authConfig` value, because if I insert these lines below 99:

````
      else
        opts.authconfig = authConfigFn('https://index.docker.io/v1/')
````

everything seems to work.

I suspect that the registry value from the Galleyfile `CONFIG` is not being picked up and that may be related, but I'm not sure.

Any ideas?

**Comment by phopkins** [comment]
Ah, that doesn't surprise me. I've never used auth'd bits on the Docker Hub.

If you put together a PR, we could get it merged and released.

**Comment by phopkins** [comment]
Fixed to some degree in v1.0.1. We use the default DockerHub credentials for any pulls for an image that doesn't have dots in it.
